castle/cms/commands.py

```diff
@@ -309,7 +345,28 @@ def convert_to_pdf(self, filepath, filename, output_dir):
 
 
 try:
-    docsplit = DocSplitSubProcess()
+    loffice = LibreOfficeSubProcess()
 except IOError:
-    logger.exception("No docsplit installed.")
-    docsplit = None
+    logger.exception("Libreoffice not installed. castle.cms"
+                     "will not be able to convert text files to pdf.")
+    loffice = None
+
+
+class DocSplitSubProcess(BaseSubProcess):
```

Comment:
is this still supposed to be called DocSplit subprocess? Could be moved somewhere now that we won't be using docsplit.

Reply:
I did this to provide backwards compatibility with any possible addons that have been added on by our users.

Reply:
Or to be more accurately, users that use castle.cms but doesn't use our services.